### ISSUE TITLE
Discord - Github push notifications webhook

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#1330]
# Feature Request
![https://camo.githubusercontent.com/649bc20b16a30205e8c1b8fd61c81b7f07a21993/687474703a2f2f692e696d6775722e636f6d2f41426c77544c662e706e67](https://camo.githubusercontent.com/649bc20b16a30205e8c1b8fd61c81b7f07a21993/687474703a2f2f692e696d6775722e636f6d2f41426c77544c662e706e67)

Discord can have a dedicated live notification webhook to Github. Meaning... we can make the UX of the tracker of Bforartists with Discord a bit more fusioned, so Discord users can follow tracker progress or updates automatically. 

The setup is simple enough, by making a bot and then linking it to the webhook under settings in Github (you need Admin access to do so). 

**Documentation/Tutorial:**
https://gist.github.com/jagrosh/5b1761213e33fc5b54ec7f6379034a22
https://support.discordapp.com/hc/en-us/articles/228383668-Intro-to-Webhooks

### TLDR
Can we push notifications for some key events (eg, task created, task closed) to a dedicated thread in Discord?

**Comment by ReinerBforartists** [comment]
Doesn't really speed up anything. But i have nothing to say against it. The more publicity the better :)

**Comment by ReinerBforartists** [comment]
Hm, i have a bit headache though that the numbers of people who just gives good meant tips increases while the nubmers of people who really fixes the issues stays the same. I don't really like to work under permanent pressure.

**Comment by Draise14** [comment]
Ah.. well it's to drive development more than crowd really. Didn't mean to make pressure. 

I'm still cooking up a way to free up my time and make my work more about BFA, but it's still a WIP.

**Comment by ReinerBforartists** [comment]
See, there comes the pressure :D

**Comment by ReinerBforartists** [comment]
Closing this one. The idea was nice, but as told, i would hate to work under pressure.

**Comment by Draise14** [comment]
I don't see how there is any pressure with live updates... but oh well.

**Comment by ReinerBforartists** [comment]
I can also simply stop working when that's what you want. I dislike to work in public. This is pressure. And not welcome.

**Comment by Draise14** [comment]
Woah woah, totally not what I wanted. Sorry man, didn't mean to stress you out. Sorry.

**Comment by ReinerBforartists** [comment]
No harm done, i just heavily dislike the idea after thinking a bit more about it. And in fact this strong that i would really quit then. There is no real benefit, only some disadvantage. What stands in the tracker means, the task can be fulfilled in best case, not that it must be fulfilled under all circumstances. And some tasks may raise some false hopes then. Add to this that we will surely see some questions in which timeframe this and that task is fulfilled.

The tracker is already public enough. Everybody interested can have a look here.
